readme.md

```diff
@@ -60,9 +60,11 @@ Some good apps written with Electron.
 ## Tools
 
 - [electron-packager](https://github.com/maxogden/electron-packager) - Package and distribute your Electron app for OS X, Linux and Windows (.app, .exe, etc).
+- [electron-builder](https://github.com/loopline-systems/electron-builder) - Build installers for Windows and OS X from Windows, Linux, or OS X.
```

Comment:
:+1:  They can read about the details of platform support on project page itself.